logos/src/logos/main.py

```
@@ -3592,6 +3595,101 @@ async def delete_user(user_id: int, request: Request):
+        caller = db.get_user_by_logos_key(logos_key)
+        if caller["role"] == "app_admin" and not db.is_team_owner(team_id, caller["id"]):
+            raise HTTPException(status_code=403, detail="You do not own this team")
+        team = db.get_team(team_id)
+        if team is None:
+            raise HTTPException(status_code=404, detail="Team not found")
+        members = db.list_team_members(team_id)
+    return {"team": team, "members": members}
+
+
+@app.post("/teams/{team_id}/members", tags=["teams"])
+async def add_team_member(team_id: int, body: AddTeamMemberRequest, request: Request):
```

Flake8 (7.3.0)

[error] 'AddTeamMemberRequest' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)